requirements-lint.txt

```diff
@@ -0,0 +1,3 @@
+autopep8
```

Comment:
No new top level files, in-line in setup.py

Reply:
Can't in-line, need to be able to install lint dependencies separately for separate lint ci. I am open to better suggestions, but I'm trying to avoid having to install 40 dependencies (including jupyterlab) just to lint